packages/presentation/pubspec.yaml

```diff
@@ -20,4 +20,5 @@ dev_dependencies:
   flutter_lints: ^2.0.0
 
 flutter:
-
+  assets:
+    - assets/
```

Comment:
usually assets(fonts, icons, images, strings) located in root folder of your app (at least in my experience)
@sergei-mikhailovskii-idf correct me if I'm wrong

Reply:
we did it in the root pubspec, yes  but it's up to u

Reply:
Fixed